pull.py

```diff
@@ -0,0 +1,41 @@
+import pandas as pd
+import requests
+import plotly.offline as py
+import plotly.graph_objs as go
+
+index = ['MU','GE','AMD','BAC']
+
+
+def main():
+    stocks = pd.DataFrame()
+    i = 0
+    while i < len(index):
+        r = requests.get('https://www.alphavantage.co/query?function=TIME_SERIES_DAILY&symbol=' + str(
+            index[i]) + '&datatype=csv&apikey=603MDIJGG9TGNV60')
+        file = open('C:\\Users\\brain\\PycharmProjects\\Stocks\\pull.csv', "w+")
```

Comment:
I think this will go away anyway by using the content.decode() but this will not work on my machine if I run your code because you have a hard-coded path. You should always use relative directories. Look into os.path.abspath(__file__) as a way to get the current directory the script is in.